src/test/java/spoon/test/processing/processors/ParallelProcessorTest.java

```diff
@@ -160,12 +158,12 @@ public void process(CtElement element) {
 				singleThreadCounter.incrementAndGet();
 			}
 		}).noClasspath(true).outputDirectory(folderFactory.newFolder()).buildModel();
-		for (int j = 0; j < atomicCounter.length(); j++) {
-			singleThreadCounter.set(singleThreadCounter.get() - atomicCounter.get(j));
-		}
-		assertTrue(singleThreadCounter.get() == 0);
+
+
+		int parallelCount = IntStream.range(0, atomicCounter.length()).map(atomicCounter::get).sum();
+		assertThat(parallelCount, equalTo(singleThreadCounter.get()));
 		// because only 3 are used
```

Comment:
maybe add a "the 4. processor must be unused" here?

Reply:
Same as above.